services/graphiql/src/test/resources/application.properties

```diff
@@ -1,4 +1,6 @@
 keycloak.auth-server-url=http://localhost:8180/auth
 keycloak.realm=activiti
 keycloak.resource=activiti
-keycloak.public-client=true
\ No newline at end of file
+keycloak.public-client=true
+
+server.servlet.context-path=/default-app
```

Comment:
@igdianov are we sure that we want to do that? 
Usually application names should be picked up from spring.application.name instead of playing with the servlet.context-path. 
I am trying to understand how this is going to work for example in community where applications are not prefixed with the path and just the services (spring.application.name) is used in the gateway. 
From what I understand I would rather do something like: 
If we deploy a new Notification Service we should give it a name to the spring boot application which will be used by the gateway. So for example, 
<gateway>/default-notification-service/graphql will be the endpoint if we deploy an instance of the service,. For our example deployments we can name this service default-notification-service, because we are deploying a single application in our acceptance tests. 
In this way we keep consistency and we don't play with the context.. does this make sense?

Reply:
@salaboy It is needed to support context path based release naming used in APS that exposes ingresses for `/graphql`, `/ws/graphql`, `/graphiql` endpoints to avoid using `rewrite-target` annotation, i.e. when application chart is deployed in APS, it uses release name, i.e. `default-app` to expose all application ingresses on the same domain using path like `/default-app/graphql` naming scheme. 

By default, in community, the servlet context and ingress path is using `/` root. In APS, application chart the context path and ingress is using `/{{ .Release.Name }}` template value to override the context as needed.